pom.xml

```diff
@@ -43,8 +43,8 @@
 
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <maven.compiler.source>1.8</maven.compiler.source>
-        <maven.compiler.target>1.8</maven.compiler.target>
+        <maven.compiler.source>11</maven.compiler.source>
```

Comment:
you can keep target compatibility to 1.8 here, the only thing we need effectively in this pull request is the upgrade of Guava.
Any chance to get that merged?

Reply:
It's ready now! Checkout version 2.4 ;)